### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by noook**

### CODE REVIEW

**Review by Alvai — APPROVED** [review]
Seems Ok 👍

File: README.md [hunk 1]
```diff
@@ -1,7 +1,29 @@
...
+**Ex :** `neil/add-new-task-command` ou `neil/I-#13` où 13 est le numéro de l'issue correspondant à la feature.
+
+## Fusion de code
+
+**Tout code à fusionner dans `master` doit passer dans une pull request.**
+
+Les pulls requests se font à [https://github.com/DEV-2020/\<repo>/compare](https://github.com/DEV-2020/welcome/compare) où `<repo>`
+correspond au dépôt dans lequel vous souhaitez faire une pull request. **Faites attention aux branches que vous souhaitez fusionner.**
+
+Les pulls requests doivent être validées par au minimum 1 personne ayant accès au dépôt. Même si une personne a déjà donné son accord,
```
Comment (Alvai) [inline README.md]: Switch à 2 Review

**Comment by MrStrikes** [comment]
Everything is ok

**Review by MrStrikes — DISMISSED** [review]
Everything looks okay !

**Review by Alvai — CHANGES_REQUESTED** [review]
Switch le nombre de user

**Review by MrStrikes — DISMISSED** [review]

**Review by Alvai — DISMISSED** [review]

**Pull request merged**